### PR TITLE
Unit Tests

### PR DESCRIPTION
**Pull request by manojleaton**
Fixes #BLUI-5414 .

#### Changes proposed in this Pull Request:

- Renamed all the test files as React workflows
- Added small changes to screen test cases

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- yarn test

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by ektaghag-eaton — COMMENTED** [review]
Are we going to fix console error in this story?
Screenshot 2024-03-19 at 3 08 12 PM
Screenshot 2024-03-19 at 3 08 47 PM

**Comment by ArshdeepSingh3-eaton** [comment]
No Ekta
Cleanup is not a part of this story

**Comment by ArshdeepSingh3-eaton** [comment]
Testcases looks good

File: login-workflow/src/__tests__/screens/VerifyCodeScreen.test.tsx [hunk 1]
```diff
@@ -33,13 +33,11 @@ describe('Verify Code Full Screen Test cases', () => {
...
         renderer();
-
         expect(screen.getByText('Verify Email')).toBeOnTheScreen();
     });
 
     it('clicking on close Icon test', () => {
         renderer();
-
         const closeIcon = screen.getByTestId('workflow-card-icon');
         fireEvent.press(closeIcon);
```
Comment (ektaghag-eaton) [inline login-workflow/src/__tests__/screens/VerifyCodeScreen.test.tsx:42]: we are not testing whether the close function is called or not after clicking on the icon. Can you add function here to check?

File: login-workflow/src/__tests__/screens/VerifyCodeScreenBase.test.tsx [hunk 2]
```diff
@@ -1,81 +1,67 @@
...
                         onPrevious: () => jest.fn(),
                     }}
                 />
-            </SafeAreaProvider>
+            </PaperProvider>
         );
-
-        const prevButton = result.getByRole('button');
+        const prevButton = screen.getByText('Back');
         fireEvent.press(prevButton);
```
Comment (ektaghag-eaton) [inline login-workflow/src/__tests__/screens/VerifyCodeScreenBase.test.tsx:65]: Here also we need to check function is called or not after clicking on button.

File: login-workflow/src/__tests__/screens/VerifyCodeScreen.test.tsx (hunk 1 above)
Comment (manojleaton) [reply login-workflow/src/__tests__/screens/VerifyCodeScreen.test.tsx:55]: Added another test for checking close function is called or not, as it was reducing coverage percentage

**Review by ektaghag-eaton — APPROVED** [review]

**Pull request merged**